src/read_csv.py

```diff
@@ -3,22 +3,25 @@
 '''
 import urllib.request
 import pandas as pd
-'''
-fuction to fetch data.
-'''
+
 def fetch_data():
```

Comment:
Same thing here, instead of downloading from a static link, a parameter can be used to download a given dataset from github